src/SearchResults/generate-hierarchical-tree.js

```diff
@@ -17,9 +17,8 @@ function generateTrees(state) {
       state.hierarchicalFacetsRefinements[hierarchicalFacet.name][0] || '';
     var hierarchicalSeparator = state._getHierarchicalFacetSeparator(hierarchicalFacet);
     var sortBy = prepareHierarchicalFacetSortBy(state._getHierarchicalFacetSortBy(hierarchicalFacet));
-    var alwaysGetRootLevel = hierarchicalFacet.alwaysGetRootLevel;
 
-    return reduce(hierarchicalFacetResult, generateHierarchicalTree(sortBy, hierarchicalSeparator, hierarchicalFacetRefinement, alwaysGetRootLevel), {
+    return reduce(hierarchicalFacetResult, generateHierarchicalTree(sortBy, hierarchicalSeparator, hierarchicalFacetRefinement), {
```

Comment:
`generateHierarchicalTree(sortBy, hierarchicalSeparator, hierarchicalFacetRefinement)` could be put in a variable to ease readability and shorten line length?